### PR TITLE
[DOCS-6909] - Correct code examples

### PR DESCRIPTION
**Pull request by oliveralfresco2018**

### CODE REVIEW

File: search-enterprise/3.0/admin/index.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ $ java -jar alfresco-elasticsearch-reindexing-3.0.0-app.jar \
 --spring.datasource.username=alfresco \
 --spring.datasource.password=alfresco \
 --spring.activemq.broker-url=tcp://localhost:61616?jms.useAsyncSend=true \
---alfresco.reindex.prefixes-file=file:reindex.prefixes-file.json
+--alfresco.reindex.prefixes-file=file:reindex.prefixes-file.json \
```
Comment (mjstanford) [inline search-enterprise/3.0/admin/index.md:61]: This change was not requested

File: search-enterprise/3.0/admin/index.md [hunk 2]
```diff
@@ -88,7 +88,7 @@ $ java -jar alfresco-elasticsearch-reindexing-3.0.0-app.jar \
...
```
Comment (mjstanford) [inline search-enterprise/3.0/admin/index.md:91]: This change was not requested

File: search-enterprise/latest/admin/index.md [hunk 3]
```diff
@@ -58,7 +58,7 @@ $ java -jar alfresco-elasticsearch-reindexing-3.1.0-app.jar \
 --spring.datasource.username=alfresco \
 --spring.datasource.password=alfresco \
 --spring.activemq.broker-url=tcp://localhost:61616?jms.useAsyncSend=true \
---alfresco.reindex.prefixes-file=file:reindex.prefixes-file.json
+--alfresco.reindex.prefixes-file=file:reindex.prefixes-file.json \
```
Comment (mjstanford) [inline search-enterprise/latest/admin/index.md:61]: This change was not requested

File: search-enterprise/latest/admin/index.md [hunk 4]
```diff
@@ -88,7 +88,7 @@ $ java -jar alfresco-elasticsearch-reindexing-3.1.0-app.jar \
...
```
Comment (mjstanford) [inline search-enterprise/latest/admin/index.md:91]: This change was not requested

File: search-enterprise/3.0/admin/index.md (hunk 1 above)
Comment (oliveralfresco2018) [reply search-enterprise/3.0/admin/index.md]: Fixed for all

**Review by mjstanford — APPROVED** [review]

**Review by anxumalo — APPROVED** [review]
LGTM

**Pull request merged**